pkg/logger/config.go

```diff
@@ -0,0 +1,56 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// Config helps to construct a customized zap logger.
+type Config struct {
```

Comment:
does this need to be exported?

Reply:
If it's not needed elsewhere, I would make it unexported and figure something else for the tests
Let's sync up offline